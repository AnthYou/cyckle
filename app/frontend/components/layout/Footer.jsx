import React from "react";

const Footer = () => {
  return (
    <footer className="p-8 bg-gray-100">
      <p className="text-center text-lg font-semibold mb-0">
        Cyckle
      </p>
    </footer>
  )
}

export default Footer;
