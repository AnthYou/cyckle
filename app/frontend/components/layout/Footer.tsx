import React from "react";

import classes from "./Footer.module.scss";

const Footer = () => {
  return (
    <footer className={`${classes.Footer}`}>
      <div className="container flex flex-col text-center sm:flex-row sm:text-left">
        <div className={classes.logo}>
          <h3>Cyckle</h3>
          <p>Location de vélos entre particuliers</p>
        </div>
        <div className="flex flex-1 justify-between">
          <ul className="sm:mr-4">
            <li>S'inscrire / se connecter</li>
            <li>Blog</li>
            <li>Nous rejoindre</li>
          </ul>
          <ul>
            <li>Contactez-nous</li>
            <li>F.A.Q.</li>
            <li>Politique de confidentialité</li>
            <li>Mentions légales</li>
          </ul>
        </div>
      </div>
    </footer>
  );
};

export default Footer;
