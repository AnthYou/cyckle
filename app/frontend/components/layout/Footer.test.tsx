import React from "react";
import renderer from "react-test-renderer";

import Footer from "./Footer";

describe('<Footer />', () => {
  it('should render correctly', () => {
    const component = renderer.create(<Footer />);
    const tree = component.toJSON();

    expect(tree).toMatchSnapshot();
  })
});
