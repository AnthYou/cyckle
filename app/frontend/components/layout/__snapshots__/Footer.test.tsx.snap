// Vitest Snapshot v1

exports[`<Footer /> > should render correctly 1`] = `
<footer
  className="_Footer_ff693a"
>
  <div
    className="container flex flex-col text-center sm:flex-row sm:text-left"
  >
    <div
      className="_logo_ff693a"
    >
      <h3>
        Cyckle
      </h3>
      <p>
        Location de vélos entre particuliers
      </p>
    </div>
    <div
      className="flex flex-1 justify-between"
    >
      <ul
        className="sm:mr-4"
      >
        <li>
          S'inscrire / se connecter
        </li>
        <li>
          Blog
        </li>
        <li>
          Nous rejoindre
        </li>
      </ul>
      <ul>
        <li>
          Contactez-nous
        </li>
        <li>
          F.A.Q.
        </li>
        <li>
          Politique de confidentialité
        </li>
        <li>
          Mentions légales
        </li>
      </ul>
    </div>
  </div>
</footer>
`;
